executors/icu4j/74/executor-icu4j/src/main/java/org/unicode/conformance/testtype/langnames/LangNamesTester.java

```diff
@@ -24,6 +24,12 @@ public ITestTypeInputJson inputMapToJson(Map<String, Object> inputMapData) {
     result.language_label = (String) inputMapData.get("language_label", null);
     result.locale_label = (String) inputMapData.get("locale_label", null);
 
+    String lang_display_string = (String) inputMapData.get("languageDisplay", null);
+    if (lang_display_string == null || lang_display_string.equals("standard")) {
```

Comment:
Instead of keeping this is as a `String` and doing a string comparison, you should use the string->enum parsing function that you created in the enum class. That function also defines and handles the default.

Reply:
Done